src/components/home-container/index.jsx

```diff
@@ -5,42 +5,51 @@ import Button from "components/base/button";
 import CreateTask from "components/task/create-new";
 import TaskCard from "components/task/existing-task/task-card";
 import FilterBtnContainer from "components/filter-btn-container";
+import NoTaskFound from "components/not-found/task";
 
 function HomeContainer() {
   const tasks = useSelector((state) => state.todo.tasks);
   const [isNewTaskRequested, setIsNewTaskRequested] = useState(false);
 
-  const handleSuccessfulTaskEntry = () => setIsNewTaskRequested(false);
-  const handleCreateBtnClick = () => setIsNewTaskRequested(true);
-
   return (
     <div className={`container mx-auto ${styles.homeWrapper}`}>
       <h1>Add Tasks</h1>
       <div className={`flex justify-between ${styles.buttonContainer}`}>
-        <Button onButtonClick={handleCreateBtnClick}>Create</Button>
+        <Button onButtonClick={() => setIsNewTaskRequested(true)}>
+          Create
+        </Button>
 
         <FilterBtnContainer />
       </div>
 
       <div className="grid grid-cols-3 gap-34px">
         {isNewTaskRequested && (
-          <CreateTask onSuccessfullTaskEntry={handleSuccessfulTaskEntry} />
+          <CreateTask
+            onSuccessfullTaskEntry={() => setIsNewTaskRequested(false)}
+            onDeleteBtnClick={() => setIsNewTaskRequested(false)}
+          />
         )}
 
-        {tasks.map((task) => (
-          <TaskCard
-            key={task.id}
-            title={task.title}
-            createdAt={task.createdAt}
-            isCompleted={task.isCompleted}
-            completedAt={task.completedAt}
-          />
-        ))}
+        {tasks.length > 0 &&
+          tasks.map((task) => (
+            <TaskCard
+              key={task.id}
+              taskId={task.id}
+              title={task.title}
+              createdAt={task.createdAt}
+              isCompleted={task.isCompleted}
+              completedAt={task.completedAt}
+            />
+          ))}
       </div>
 
-      <div className="flex justify-center">
-        <Button className={styles.loadMoreBtn}>Load More</Button>
-      </div>
+      {tasks.length <= 0 ? (
+        <NoTaskFound />
+      ) : (
+        <div className="flex justify-center">
+          <Button className={styles.loadMoreBtn}>Load More</Button>
+        </div>
```

Comment:
We will not always show Load more button. We will have separate logic for that.
```suggestion
      {taskAvailable === false && <NoTaskFound /> }
      
      {ShowLoadMoreButton && 
      <div className="flex justify-center">
          <Button className={styles.loadMoreBtn}>Load More</Button>
        </div>
        }
```

Reply:
yes vai , I have added the pagination logic in [step-6 add pagination PR](https://github.com/azo-wtag/todo-app-react/pull/5) . As pagination button will only be visible if no of task count greater than zero that's why put it inside `if-else` condition but added the logic later on respective [PR](https://github.com/azo-wtag/todo-app-react/pull/5). @hhu-wtag vai